_posts/admin/2022-01-27-clusterproxy.md

```diff
@@ -111,10 +111,10 @@ public:
     uploadEndpoint: 'https://bbb-01.example.com/bigbluebutton/presentation/upload'
 ```
 
-Create the these unit file overrides:
+Create (or edit if it already exists) these unit file overrides:
 
-* `/etc/systemd/system/bbb-html5-frontend@.service.d/cluster.conf`
-* `/etc/systemd/system/bbb-html5-backend@.service.d/cluster.conf`
+* `/usr/lib/systemd/system/bbb-html5-frontend@.service`
+* `/usr/lib/systemd/system/bbb-html5-backend@.service`
```

Comment:
Overriding systemd units should be kept in /etc/systemd, it is not necessary  to touch package-owned files in /usr/lib/systemd.

Reply:
Ohh, thanks for the suggestion, I just corrected that way because it made sense to me at the time and also worked when I did that... Could you expand a little more on this override thing?

Reply:
systemd config in `/lib/systemd` and `/usr/lib/systemd` is meant to be managed by packages. `/etc/systemd` is the systems administrators domain.